webpack.config.client.js

```diff
@@ -109,6 +110,10 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
           );
         },
       ),
+      new webpack.ContextReplacementPlugin(/moment[\\]locale$/, /en/),
```

Comment:
We need to keep our webpack config really, really well commented!

Can you add line-by-line inline comments explaining why all of this is needed please.